packages/telemetry/src/next.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getApp } from '@firebase/app';
+import { captureError, getTelemetry } from './api';
+import { type Instrumentation } from 'next';
+
+/**
+ * Automatically report uncaught errors from server routes to Firebase Telemetry.
+ *
+ * @example
+ * ```javascript
+ * // In instrumentation.ts (https://nextjs.org/docs/app/guides/instrumentation):
+ * import { nextOnRequestError } from '@firebase/telemetry/next';
```

Comment:
Update path since we moved packages, also no `@` in public-facing code samples.

Also I think you can do it in one line as `export { nextOnRequestError as onRequestError }  from 'firebase/telemetry'`